### DISCUSSION TITLE
proprietary codecs NOT WORKING

### DISCUSSION THREAD

**Discussion by MarcellVokk** [cefsharp/CefSharp#3316] [General]
OK, so I'm so outraged right now! I can not play a YOUTUBE video on the browser, because it needs HTML5 video or shomething like that... I have tryed building it on my own, but I just could not get my head around it. And it also took a full night to download it from github! Please fix it or shomething because idk what to do!!!

**Comment by amaitland** [comment]
Please confirm you have read https://github.com/cefsharp/CefSharp/wiki/General-Usage#multimedia-audiovideo

Legally to distribute builds with support for `Proprietary Codecs (H264/AAC)` I would have to pay licensing fees. This would cost me tens of thousands of dollars as it's a fee per user license. 

If you are having problems building the `Chromium embedded framework(CEF)` with `Proprietary Codec` support then https://magpcss.org/ceforum/index.php is the place to ask for help. See https://magpcss.org/ceforum/viewtopic.php?f=6&t=13515 for the most recent discussion that I'm aware of (there are many pages to the thread).

---
I'm reliably informed that the new `WebView2` browser control from `Microsoft` supports `Proprietary Codecs`, perhaps is a better fit for you needs.
